.github/workflows/canary-integration-test.yml

```diff
@@ -11,6 +11,9 @@ on:
       - master
       - release-*
 
+env:
+  CEPH_ARGS: --name client.rookoperator
```

Comment:
In the `CreateOrLoadClusterInfo()` method, an important update is that in the `rook-ceph-mon` secret
- The default for the CephUsername would be `rookoperator`
- Generate a new username/keyring combination to store in that secret for the toolbox. 
- For upgraded clusters, if the client.admin key is found, we need to generate the rookoperator user and replace it in the secret.

Reply:
> Reverting all these arguments will be important. New parameters to toolbox commands shouldn't be necessary.
> 
> * All operator commands should internally use the client.rookoperator
> * The toolbox config should internally use the client.rooktoolbox
> 
> Likely the code changes for this PR won't be a lot, we just need to update a few critical places.

I'll add  the  changes soon, I was trying to make ci green

Reply:
> This shouldn't be necessary for the CI to pass the cephx name. If it is needed, this means anybody using the toolbox would be burdened with passing it, which we need to avoid. Anything using the toolbox should just inherit the default ceph.conf that has that line in the ceph.conf with the name: `[client.rookoperator]`

the config are being used but still  it fail without `--name`
```
 exec -it rook-ceph-tools-97994c8f6-mn4l8 -- sh
sh-4.4$ echo $ROOK_CEPH_USERNAME
client.rookoperator
sh-4.4$ echo $ROOK_CEPH_SECRET
AQAvhtZi0C7NIhAAwsDG8LQgfJFs/brVVUz+rg==
sh-4.4$ ceph status
[errno 1] RADOS permission error (error connecting to the cluster)```

Reply:
```
cat /etc/ceph/ceph.conf 
[global]
mon_host = 10.100.7.218:6789

[client.rookoperator]
keyring = /etc/ceph/keyring
```
this is after I edit the script inside toolbox.yaml